frontend/src/components/details-car-panel/details-car-panel.tsx

```diff
@@ -0,0 +1,194 @@
+import { FC } from 'react';
+
+import {
+  WishlistInput,
+  DeleteWishlistInput,
+} from '@autoline/shared/common/types/types';
+import { DetailsCarPanelPropsType } from '@common/types/types';
+import { HeartIcon } from '@components/common/icons/icons';
+import { convertPrice } from '@helpers/utils/convert-price';
+import {
+  useGetComplectationsForPanelQuery,
+  useGetRateQuery,
+} from '@store/queries/details-panel';
+import {
+  useCreateWishlistMutation,
+  useDeleteWishlistMutation,
+} from '@store/queries/preferences/wishlist';
+import { clsx } from 'clsx';
+
+import styles from './styles.module.scss';
+
+const DetailsCarPanel: FC<DetailsCarPanelPropsType> = ({
+  complectationId = '',
+  modelId = '',
+}) => {
+  const { data, isLoading } = useGetComplectationsForPanelQuery({
+    complectationId,
+    modelId,
+  });
+  const { data: rate } = useGetRateQuery();
+  const [createWishlist] = useCreateWishlistMutation();
+  const [deleteWishlist] = useDeleteWishlistMutation();
+
+  const handleCreateWishlist = async (): Promise<void> => {
+    const data: WishlistInput =
+      complectationId === '' ? { modelId } : { complectationId };
```

Comment:
`''` is already considered nullish value, so it can be simplified to the `complectationId ? { complectationId } : { modelId }`